packages/styles/forms.css

```
@@ -159,10 +161,12 @@ textarea.Field--has-error:focus:hover,
 
 .Error {
   color: var(--error);
+  border-top: 1px solid var(--error);
   text-align: left;
   font-size: var(--text-size-smallest);
-  font-weight: var(--font-weight-normal);
-  padding-left: var(--space-half);
+  margin-top: var(--space-half);
+  margin-left: calc(24px + 2px + var(--space-half));
```

I know this was copy pasta, but we should update this to use `--icon-size`:

```suggestion
  margin-left: calc(var(--icon-size) + 2px + var(--space-half));
```